browser/resources/settings/brave_leo_assistant_page/model_config_ui.html

```diff
@@ -174,6 +174,36 @@
         error-message="[[invalidUrlErrorMessage]]"
       ></cr-input>
     </div>
+    <div class="input-container">
+      <div class="custom-label">
+        <div class="label-text">
+          <span>$i18n{braveLeoAssistantInputContextSizeLabel}</span>
+          <leo-tooltip mode="default">
+            <div
+              slot="content"
+              class="tooltip-content"
+            >
+              $i18n{braveLeoAssistantInputContextSizeTooltipInfo}
+            </div>
+            <leo-button
+              size="small"
+              kind="plain-faint"
+              fab
+            >
+              <cr-icon class="small" icon="info-outline"></cr-icon>
+            </leo-button>
+          </leo-tooltip>
+        </div>
+      </div>
+      <div class="note" inner-h-t-m-l="[[setDefaultContextSize_('braveLeoAssistantInputContextSizeProxyNote')]]"></div>
```

Comment:
Do we need to use `inner-h-t-m-l` here? The string doesn't seem like it contains any html, and it might be a security vulnerability if we get a malicious translation.

```suggestion
<div class="note">[[setDefaultContextSize_('braveLeoAssistantInputContextSizeProxyNote')]]</div>
```

Reply:
Good point! I needed to do a substitution, and went searching for a way to do this with $i18n. Found an example (the only example I could locate) in the Chromium codebase which showed it, and they leveraged the `inner-h-t-m-l` attribute.

I noticed when I passed a second argument through the attribute route, I would get TrustedHTML errors. This didn't bother me much though, since I opted to handle the replacement logic within the `setDefaultContextSize` method itself. Curiously, with only a single argument passed to that method, I received no warning regarding the use of this attribute.

All that aside, I can't believe I missed this. Had tunnel vision 😂